model/github.py

```diff
@@ -169,6 +172,9 @@ def set_auth_token(self, auth_token):
     def private_key(self):
         return self.raw.get('privateKey')
 
+    def set_key(self, key: str):
+        self.raw['privateKey'] = key
```

Comment:
hmmm..... I am not too sure about that tbh :-/
-> all other model-classes only have readonly-methods - this would be the very first one that deviates from that :-/

Reply:
Done, albeit at the price of direct access to the raw config in the added module instead. I'd have expected us to add more setters if/when we add more rotation code, but we can also revisit the issue then.